hosted/test-vectors/protocol/vectors/parse-close.json

```diff
@@ -1,20 +1,20 @@
 {
   "description": "Close parses from string",
-  "input": "{\"metadata\":{\"from\":\"did:dht:ese8j947ro8uth1mqpnhtcr64s39zn1tgq75q34uo6hikf7zzmyy\",\"to\":\"did:dht:n46hom5afi6xrsxmddx5rjecyyx1faz4ocs4ie43tfkyo4darh9y\",\"exchangeId\":\"rfq_01hrqn6pk2ehasa0p9d14hsp8s\",\"protocol\":\"1.0\",\"kind\":\"close\",\"id\":\"close_01hrqn6pk2ehasa0p9d6wtgqkr\",\"createdAt\":\"2024-03-11T21:02:55.586Z\"},\"data\":{\"reason\":\"The reason for closing the exchange\"},\"signature\":\"eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDpkaHQ6ZXNlOGo5NDdybzh1dGgxbXFwbmh0Y3I2NHMzOXpuMXRncTc1cTM0dW82aGlrZjd6em15eSMwIn0..TL3H01A0J0Iz3404lxHviua4SCIIBm4z_7N8N287tr3nZbbXU8djqDS1UPUUPztt-8wvgWTBS-oPOneYCoqLCA\"}",
```

Comment:
why are we updating the existing vectors? we should avoid updating existing vectors unless we need to introduce a breaking change.